www/src/templates/component-page-template.tsx

```diff
@@ -137,10 +140,22 @@ export default function PageTemplate({
             <p className="small mb-0">{mdx.frontmatter.notes}</p>
           </Alert>
         )}
-        <div className="d-flex justify-content-between align-items-start">
-          <h1 className="mb-4">{mdx.frontmatter.title}</h1>
-          <LeaveFeedback />
-        </div>
+        <Stack
+          className="justify-content-between"
+          direction={!isMobile && 'horizontal'}
+        >
+          <h1 className={isMobile ? 'mb-2' : 'mb-4'}>
+            {mdx.frontmatter.title}
+          </h1>
+          <Stack
+            className="mb-4"
+            direction={!isMobile && 'horizontal'}
+            gap={3}
```

Comment:
[suggestion] I might recommend using  `gap={2}` to have them slightly closer together.

Reply:
Corrected